modules/consentManagement.js

```diff
@@ -374,9 +556,20 @@ export function setConsentConfig(config) {
       utils.logError(`consentManagement config with cmpApi: 'static' did not specify consentData. No consents will be available to adapters.`);
     }
   }
-  if (!addedConsentHook) {
+
+  if (consentModule === 'usp' && !addedConsentHookUSP) {
+    $$PREBID_GLOBAL$$.requestBids.before(requestCcpaBidsHook, 50);
+    addedConsentHookUSP = true;
+  }
+
+  if (!addedConsentHook && consentModule === 'gdpr') {
     $$PREBID_GLOBAL$$.requestBids.before(requestBidsHook, 50);
+    addedConsentHook = true;
   }
```

Comment:
I believe this will run these `synchronously`.  We want them to run `asynchronously`.  We don't want to wait for the `gdpr` api to finish before calling the `usp` api (or vice-versa).

Reply:
Technically I am calling this method for each time I loop through the array set I build at the bottom of the page. They should be running async.

Reply:
ok, I thought the hooks api did these synchronously, but Isaac said the requestBids hook is configured to be asynchronous.